pom.xml

```diff
@@ -146,6 +146,16 @@
 							<directory>${project.build.directory}</directory>
 							<include>${project.build.finalName}.jar</include>
 						</resource>
+
+						<!-- TDS_Student requires file-system test definitions.  The resource directories below must
+							be defined in the build machine's local file-system at the specified paths
+							to be embedded into the student docker container image.
+							Modify as required.-->
+						<resource>
+							<targetPath>/</targetPath>
+							<directory>${project.basedir}/src/main/resources/bank</directory>
```

Comment:
If I wanted to build this project and include resources would I have to modify this directory or can I override it with an argument?